src/mongocxx/test/database.cpp

```diff
@@ -36,9 +36,7 @@ namespace {
 
 // Clean up the given collection if it exists
 void clear_collection(const database& database, bsoncxx::stdx::string_view collection_name) {
-    if (database.has_collection(collection_name)) {
-        database[collection_name].drop();
-    }
+    database[collection_name].drop();
```

Comment:
Instead of changing this method, can we just remove it now and replace the calls to `clear_collection()` with `database[collection_name].drop()` ?